java/overrides.js

```diff
@@ -50,6 +53,10 @@
     'appium android check window': {skip: true}, //wrong ignore region ticket 2396
     'appium android check region with ignore region': {skip: true}, //wrong ignore region region ticket 2396
     'appium iOS check region with ignore region': {skip: true}, //wrong ignore region region ticket 2396
+	
+	'check window fully on page with horizontal scroll with css stitching': {skip: true}, //Could not start a new session
+	'check window fully on page with horizontal scroll with scroll stitching': {skip: true}, //Could not start a new session
+	'appium android check region': {skip: true}, //InvalidSelectorException: Locator Strategy 'class' is not supported for this session
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"appium android check region": {skip: true}, //InvalidSelectorException: Locator Strategy 'class' is not supported for this session
```